README.md

```diff
@@ -1,3 +1,273 @@
+    credentials:                              # credentials for the dns provider
+
+  <a href="#landscapeidentity">identity</a>:
+    users:
+      - email:                                # email (used for Gardener dashboard login)
+        username:                             # username (displayed in Gardener dashboard)
+        password:                             # clear-text password (used for Gardener dashboard login)
+      - email:                                # see above
+        username:                             # see above
+        hash:                                 # bcrypted hash of password, see above
+</pre>
+
+
+### landscape.name
+```yaml
+landscape:
+  name: <URL-friendly name>                       # Gardener managed k8s landscape name
+```
+Arbitrary name for your landscape. Will partly be used for naming of resources, for example, the etcd buckets.
+
+### landscape.cluster
+```yaml
+cluster:                                            # Information about your base cluster
+  kubeconfig: <relative path + filename>            # Path to your `kubeconfig` file, relative to folder `landscape`   
+  domain: <prefix>.<cluster domain>                 # Unique basis domain for DNS entries
+  iaas: <gcp|aws|azure>                             # IaaS provider (coming soon: openstack|alicloud)
+  region: <major region>-<minor region>             # Example: europe-west1
+```
+
+Information about your base cluster, where the Gardener will be deployed on.
+
+| Field | Type | Description | Example |
+|:------|:--------|:--------|:--------|
+|`kubeconfig`|File path| Path to your kubeconfig, relative to your landscape folder. It is recommended to create a kubeconfig file in your landscape folder to be able to sync all files relevant for your installation with a git repository.| `./kubeconfig` |
+|`domain`| Unique name| Basis domain for DNS entries. As a best practice, use an individual prefix together with the cluster domain of your base cluster.|`vedge.gcp.dev.k8s.jacksgrocerystore.com`|
+|`iaas`| Fixed value | IaaS provider you would like to install Gardener on. | `gcp` |
+|`region`|IaaS provider specific| Region where your Kubernetes base cluster is deployed. | `europe-west1` (GCP)|
+
+### landscape.networks
+```yaml
+networks:                                 # CIDR IP ranges of base cluster
+  nodes: <CIDR IP range>                                  
+  pods: <CIDR IP range>                                   
+  services: <CIDR IP range>
+```
+CIDR ranges of your base cluster.
+
+Finding out CIDR ranges of your cluster is not trivial. For example, GKE only tells you a "pod address range" which is actually a combination of pod and service CIDR. However, since the `kubernetes` service typically has the first IP of the service IP range and most methods to get a kubernetes cluster tell you at least something about the CIDRs, it is usually possible to find out the CIDRs with a little bit of educated guessing.
```

Comment:
It took us a while to find the correct CIDRs for Azure. So maybe, we could extend the documentation here?
For AKS, one can retrieve the node CIDR using 
`az network vnet list -g MC_${CLUSTER_NAME}_${CLUSTER_NAME}_$REGION -o json | jq -r '.[] | .subnets[] | .addressPrefix'` 
and the pod CIDR with
`kubectl -n kube-system get daemonset/kube-proxy -o yaml | grep cluster-cidr= | grep -v annotations | cut -d = -f2'`.
As a service CIDR we used `10.0.0.0/16`
Do you plan to include similar commands for AWS or GCP?

Reply:
Thank you for your feedback. I agree that this is the tricky part.

For the first roll-out, it is not planned yet to provide these details. It may be useful to keep provider specific parts of the configuration in a separate document (tutorials?).